cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -244,11 +244,18 @@ func (r *KubeArchiveConfigReconciler) desiredRoleBinding(kaconfig *kubearchivev1
 			Kind:     "Role",
 			Name:     kaconfig.Name,
 		},
-		Subjects: []rbacv1.Subject{{
-			Kind:      "ServiceAccount",
-			Name:      kaconfig.Name,
-			Namespace: kaconfig.Namespace,
-		}},
+		Subjects: []rbacv1.Subject{
```

Comment:
I think this is OK for now as I believe you are pressed for time, but I think we should have different roles and rolebindings for this.  I know it's just one verb, but we're now giving the operator delete permission in every k9e-monitored namespace.

Reply:
I'll open an issue to address this

Reply:
https://github.com/kubearchive/kubearchive/issues/219